codemods/utils/process-element.ts

```diff
@@ -352,3 +344,11 @@ const nodePathToString = nodePath => {
   }
   return 'And error occurred';
 };
+
+// tries to grab the type parameter from `styled.div<foo>` if it can
+function getTypeParameter(nodePath: any, j: JSCodeshift) {
+  const typeParameters = nodePath.node.typeParameters;
+  const typeRef = typeParameters?.params[0];
+
+  return typeRef;
+}
```

Comment:
This file is the only code change.

Everything else is fixing tests as it related to the types